src/CMakeLists.txt

```diff
@@ -763,7 +783,7 @@ ENDIF (WANT_INTERNAL_LOKI)
 # prevent opening the cmd.exe window
 IF (WIN32)
   LIST(APPEND TORA_LIBS ${WINDOWS_LIB})
-  LIST(APPEND TORA_LIBS "ws2_32.lib")
+  #LIST(APPEND TORA_LIBS "ws2_32.lib")
```

Comment:
ws2_32.lib is windows library shipped with MSVC. The idea behind is debug builds will have "dos console" window opened to show some logs. Release build will have it hidden.
PS: let's create a separate branch for qt5 port.